Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -84,3 +84,65 @@ Deploy application
     cd "BlueprintBaseName.1/src/BlueprintBaseName.1"
     dotnet lambda deploy-serverless
 ```
+
+## Lambda Annotations
+This template now uses Lambda Annotations to bridge the gap between the Lambda programming model and a more idiomatic .NET model.
```

Comment:
Call out that it handles syncing with CloudFormation template.

Reply:
Added a line.